index.html

```diff
@@ -0,0 +1,236 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8ecd2bb40c.js"></script>
+    <title>Your Career in Web Development Starts Here | The Odin Project</title>
+  </head>
+
+  <body>
+    <nav class="bg-white">
+      <div class="navbar-brand">
+        <img src="./images/logo.png" style="height: 70px; width: 200px;" alt="">
+      </div>
+      
+      <i class="fas fa-bars fa-lg menu" style="padding-right: 16px; color: #6ABFC3;"></i>
+      
+      <div class="nav-link">
+        <ul>
+          <li>
+            <a href="#">Curriculum</a>
+          </li>
+          <li>
+            <a href="#">Community</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li style="border-right: 1px solid #828282;">
+            <a href="#">FAQ</a>
+          </li>
+          <li>
+            <a href="#">Sign Up</a>
+          </li>
+          <li>
+            <a href="#">Log In</a>
+          </li>
+        </ul>
+      </div>
+      
+    </nav>
+
+    <main class="bg-white">
+      <h3 style="color: #cc9543; font-weight: lighter;">Your Career in Web Development Starts Here</h3>
+      <p style="margin-bottom: 3.5rem;">Our full stack curriculum is free and supported by a passionate open source community.</p>
+      <div class="btn btn-blue">View Full Curriculum</div>
+      <img src="./images/main.svg" style="padding: 48px 0;" alt="">
```

Comment:
No inline css

**`[REQUIRED]`**
Add  **Padding** and **Margin** css in your framework ?